### ISSUE TITLE
Improve indication that a file has edits

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#237429]
At the moment, it's not clear to a screen reader if a file has edits. 
 
1. Enable a screen reader
2. Have some editors open
3. Apply some edits
4. Switch between tabs, see if you can tell which files have edits 
5. 🐛 there's no indication at the moment.

Similar to the `working tree` descriptor we add to the editor's title, I think we should say (Edited) in the title here so it's clear, vs using this square with a dot as an indicator. I also think this would help sighted users and beginners to understand the scenario.

<img width="610" alt="Image" src="https://github.com/user-attachments/assets/4ab11655-58c1-4944-9401-66e10348b93f" />

https://github.com/user-attachments/assets/1bd32326-2039-4af0-92a8-722a6851d27a

cc @isidorn

**Comment by isidorn** [comment]
I like the suggestion of updating the suffix of Aria Label of the editor when there are Copilot Edits present.

**Comment by jrieken** [comment]
@meganrogge We use the "normal" file decoration service for this, e.g the one that's used for SCM decorations and diagnostics. Should we have an accessibility solution that is unified across them?

**Comment by meganrogge** [comment]
I'm seeing that this never gets hit when I focus the chat editor 

https://github.com/microsoft/vscode/blob/1853994a588186cb0244cfc996550f06e9daf5c5/src/vs/workbench/contrib/chat/browser/chatEditorInput.ts#L86-L88

That should be getting called from here

https://github.com/microsoft/vscode/blob/1853994a588186cb0244cfc996550f06e9daf5c5/src/vs/workbench/browser/editor.ts#L276

But it's getting recognized as a `FileEditorInput` vs a `ChatEditorInput`

<img width="753" alt="Image" src="https://github.com/user-attachments/assets/74ccf0c6-e6e9-4b17-8106-eaabb90d25fe" />

**Comment by meganrogge** [comment]
I looked into doing this but worry it will look too crowded to include `(Edited)` in the tab's title after the name of the file. 

I'm going in a different direction here - similar to what we do for diff editors and editors with comments, I'll specify that there are pending changes in the a11y help dialog.

**Comment by jrieken** [comment]
I meant updating the aria-label with whatever the decoration's titles are and my comment was about using the file decorations infrastructure and not doing a one-off solution for this

**Comment by meganrogge** [comment]
That will make the aria label very verbose, which is something users already complain about.

**Comment by jrieken** [comment]
re https://github.com/microsoft/vscode/issues/237429#issuecomment-2578212129 - looks like we already do that

<img width="587" alt="Code - Insiders 2025-01-09 11 29 10" src="https://github.com/user-attachments/assets/74bf9c9f-395d-4c53-b098-c7c16e841e55" />

**Comment by meganrogge** [comment]
Yes, created https://github.com/microsoft/vscode/issues/237528 to track that not getting read. I think it's by design not being read, but we should allow users a way to trigger it.

**Comment by meganrogge** [comment]
To verify, open the editor's accessibilty help dialogue. Verify it indicates there are edits if there are any.
